January 2026 clerical fixes

ohexter opened the pull request:
- Add Oxford comma to 4a1, 4g1a, and 9p2a. 
- Rename the new 4b2a to 4b2b and move it to the correct location.
- Move 9f5a/9f5b to the correct location.
- Use "must not" rather than "may not" in 9p2a++, consistent with usage elsewhere.
- Add 5x5 to A2d1 (consistent with 4a1 & 4g1a)
- Correct typos in A7b1
- Add additional blank line before Article I, consistent with other Articles.
- Add back H1f (conflict between https://github.com/thewca/wca-regulations-january-2026/pull/40/ and https://github.com/thewca/wca-regulations-january-2026/pull/41).
- Add hyperlink to B2d in I5b2.
- Use consistent ' character throughout, not ’.
- Use " throughout, not “ & ”.

ohexter committed:
4b2a clerical fix

Rename the new 4b2a to 4b2b and move it to the correct location

ohexter committed:
Move 9f5a/9f5b

Correct order of Regulations

ohexter committed:
apostrophe character

use consistent ' character throughout, not ’

ohexter committed:
9p2a++ may not > must not

Consistent with usage elsewhere

ohexter committed:
Add blank line

Add additional blank line before Article I, consistent with other Articles

ohexter committed:
Add internal link

Add link to B2d in I5b2

oyyq99999 commented:
Some quotes in I4c2, I4c4, I4c5, I4c7, I5b2 and I5b3 are not consistent with others too. (Might be not comprehensive)

ohexter committed:
Consistent double quote character

Use " throughout, not “ * ”.

ohexter committed:
Align A2d1 with 4g1a

Ensure 5x5 is included
Oxford comma in 4g1a and 4a1

ohexter committed:
9p2a comma

ohexter committed:
Add back H1f

Resolves conflict between https://github.com/thewca/wca-regulations-january-2026/pull/40 and https://github.com/thewca/wca-regulations-january-2026/pull/41

ohexter committed:
Fix A7b1 typos

As raised in https://github.com/thewca/wca-regulations-january-2026/pull/64

KerrieJarman commented:
Approved by Board Vote
